pkg/scheduler/framework/util.go

```diff
@@ -262,3 +264,35 @@ func (nl *NodeLister) List() ([]*v1.Node, error) {
 	}
 	return nodes, nil
 }
+
+func nodeIsReady(obj *v1.Node) bool {
+	conditionMap := make(map[v1.NodeConditionType]*v1.NodeCondition)
+	NodeAllConditions := []v1.NodeConditionType{v1.NodeReady}
+	for i := range obj.Status.Conditions {
+		cond := obj.Status.Conditions[i]
+		conditionMap[cond.Type] = &cond
+	}
+
+	var status []string
+	for _, validCondition := range NodeAllConditions {
```

Comment:
I think this implementation is too complicated, When the number of nodes is large, there will be a large number of [] string expansion and copying

Reply:
Based on everyone's opinions, I have simplified it. Thank you